dlt/common/destination/reference.py

```diff
@@ -393,23 +451,20 @@ def update_stored_schema(
         return expected_update
 
     @abstractmethod
-    def start_file_load(self, table: TTableSchema, file_path: str, load_id: str) -> LoadJob:
+    def get_load_job(
```

Comment:
this is very misleading name. it does not get any jobs. it creates new jobs. please rename